Dockerfile

```diff
@@ -1,70 +1,23 @@
-FROM  mambaorg/micromamba:1.4-alpine AS min
-ENV HOME=/home/mambauser
-WORKDIR ${HOME}
-USER mambauser
-COPY min-environment.yml pyproject.toml README.rst ${HOME}/
-RUN micromamba create -f min-environment.yml -y --no-pyc \
- && micromamba clean -ayf \
- && rm -rf ${HOME}/.cache \
- && find /opt/conda/ -follow -type f -name '*.a' -delete \
- && find /opt/conda/ -follow -type f -name '*.pyc' -delete \
- && find /opt/conda/ -follow -type f -name '*.js.map' -delete  \
- && rm min-environment.yml
-COPY data/ ${HOME}/data
-COPY examples/ ${HOME}/examples
-COPY tests/ ${HOME}/tests
-COPY hydromt/ ${HOME}/hydromt
-RUN micromamba run -n hydromt pip install . --no-cache-dir --no-compile --disable-pip-version-check --no-deps\
- && micromamba clean -ayf \
- && find /opt/conda/ -follow -type f -name '*.a' -delete \
- && find /opt/conda/ -follow -type f -name '*.pyc' -delete \
- && find /opt/conda/ -follow -type f -name '*.js.map' -delete
- ENTRYPOINT [ "micromamba", "run", "-n", "hydromt" ]
- CMD ["hydromt","--models"]
+FROM debian:bookworm-slim as base
+ARG PIXIENV
+WORKDIR /home/hydromt
+RUN apt-get update && apt-get install -y curl
```

Comment:
It's interesting how micromamba made a base image and then added an extra stage to prevent the install programs to remain within the docker image: https://github.com/mamba-org/micromamba-docker/blob/main/Dockerfile

Maybe we can do something similar.

Reply:
if we can do that all the better!